packages/fluent-theme/src/icons/SendIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { SVGAttributes } from 'react';
+
+export function SendIcon(props: SVGAttributes<SVGElement>) {
+  return (
+    <svg
+      aria-hidden="true"
+      fill="currentColor"
```

Comment:
Do we need `fill` here if we have the one in the `<path>`?

Reply:
Yes, always having it on SVG root tag is a good practice